docs/linux/part.md

```diff
@@ -6,58 +6,54 @@ version being used.
 sector and write pointer position) relative to the partition start sector.
 Similarly, a zone reset operation must specify a target zone range relative to
 the partition start sector. In effect, a zoned block device partition can be
-treated exactly like a regular disk, using the partition zoned block device with
-a start sector of 0. All zone and IO operations will be executed correctly
-taking into account the partition start sector and size.
-
-For host aware zoned block device models, creating partitions using a standard
-tools such as <a href="https://gparted.org/" target="_blank">*gparted*</a> to
-create a GUID partition table will work as expected. Since well-known partition
-tools do not have zoned block device support implemented, users must manually
-align the partitions to zone boundaries to satisfy the kernel constraint. This
-alignement will not be automatically done by the partitioning tool.
-
-However, the lack of zoned block device support for partition management tools
-can result in write IO errors for host managed zoned disk models. If the zones
-to be used to store the partition table data are sequential write required
-zones, then the partition tool must be able to correctly write align the
-partition table information on disk with the write pointer position of the
-zones that will be written. For instance, with the GUID partition table format,
-if the last zone of the disk is a sequential write required zone, the secondary
-GPT header and table entries will not necessarily be writable at the end of the
-disk LBA space, that is, at the end of the last zone of the disk. While manually
-performing this alignment is possible, such a procedure would be very difficult
-and unreliable.
-
-As a result, Using the [*dm-linear*](../linux/dm.md#dm-linear) device mapper
-target to isolate smaller portions of a large host managed device is a better
-solution than partitions. Users should assume that partitions are not supported
-for host managed zoned block devices.
+treated exactly like a regular disk: the partition zoned block device is used 
+with a start sector of 0. All zone and IO operations will be executed 
+correctly, taking into account the partition start sector and size.
+
+For host-aware zoned block device models, creating partitions using standard tools like <a href="https://gparted.org" target="_blank">gparted</a> works as expected. Because these standard partitioning tools do not have zoned-block-device support implemented, users must manually align the partitions to the zone's boundaries in order to satisfy kernel constraints. This alignment is not automatically done by the partitioning tool.
+
+The lack of zoned-block device support for partition management tools
+can result in write-IO errors for host-managed zoned disk models. If the zones
+that are used to store the partition table data are sequential write-required
+zones, then the partition tool must be able correctly to write-align (1) the
```

Comment:
must be able correctly to write-align -> must be able to correctly write-align ?
I can fix this up when applying.